tests/index.js

```diff
@@ -1 +1,2 @@
-require('./find-prime');
\ No newline at end of file
+require('./find-prime');
+require('./filter-by-query.js');
```

Comment:
расширение файла не нужно указывать
```suggestion
require('./filter-by-query');
```